hosting/about-hosting-on-sia.md

```diff
@@ -19,6 +19,14 @@ layout:
 
 # About Hosting on Sia
 
+{% hint style="danger" %}
+Terminal app users must [upgrade to `hostd v2.2.0+`](http://sia.tech/host) by **July 4, 2025**.
```

Comment:
Users of all daemons need to upgrade by the allow height. Which is height `526000` on June 6th.
We shouldn't even mention the grace period or require height since it's not relevant for most people.